modules/lkqdBidAdapter.js

```diff
@@ -0,0 +1,234 @@
+import * as utils from '../src/utils.js';
```

Comment:
Can you please update this to only import the needed functions. We removed the 'utils' root in all other adapters

Reply:
Done